guidelines/releases/release-process.md

```diff
@@ -102,46 +130,39 @@ Follow these steps to create a release:
 
         ```yaml
         dir:
-        version: {rel_version}
+        version: {release_version}
         ```
 
         > **NOTE:** Replace only `develop/` so `develop/tests` becomes `tests/`.
 
-        Every component image is published with a version defined in the `RELEASE_VERSION` file stored in the `test-infra` repository on the given release branch.
+        Every component image is published with a version defined in the `RELEASE_VERSION` file stored in the `test-infra` repository on the given release branch. Test scripts for integration jobs like GKE Integration or GKE Upgrade are also loaded from the `test-infra` release branch.
 
     2. Check all `yaml` files for the following references:  
```

Comment:
So actually you replace `image: eu.gcr.io/kyma-project/develop/{IMAGE_NAME}:{SOME_SHA}` with `image: eu.gcr.io/kyma-project/{IMAGE_NAME}:{release_version}` in `installation/resources/installer-local.yaml` and In `eu.gcr.io/kyma-project/develop/installer:{image_tag}` with `eu.gcr.io/kyma-project/kyma-installer:{release_version}` in `installation/resources/installer.yaml`? Maybe we make it clearer by making two seperate points of that (ii and iii)?

Reply:
What about now? 😄